src/main/resources/avro/bdg.avdl

```diff
@@ -57,8 +57,55 @@ record Contig {
   union { null, string } species = null;
 }
 
+record Sequence {
+  union { null, string } bases;
+  union { null, string } qualities;
+}
+
+/**
+   The DNA fragment that is was targeted by the sequencer, resulting in
+   one or more reads.
+*/
+record Fragment {
+  /**
+     The name of this Fragment.
+  */
+  union { null, string } readName = null;
+
+  union { null, string } instrument;
+	union { null, string } runId;
+  /**
+     The following are Illumina specific, is it a problem?
```

Comment:
I'm not crazy about this.  Perhaps we can think of a more general way to support vendor-specific data.